pinephone/device.mk

```diff
@@ -34,3 +34,6 @@ PRODUCT_PACKAGES += \
 
 PRODUCT_PACKAGES += \
     android.hardware.vibrator@1.0-service.pinephone
+
+DEVICE_PACKAGE_OVERLAYS := \
```

Comment:
Please use `+=`, with :`=` I've got no navigation icons and lock screen appear.

Reply:
Done.